pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -212,7 +212,7 @@ def snipe(self, pokemon):
         last_position = self.bot.position[0:2]
         self.bot.heartbeat()
         self._teleport_to(pokemon)
-        catch_worker = PokemonCatchWorker(pokemon, self.bot)
+        catch_worker = PokemonCatchWorker(pokemon, self.bot, self.config)
```

Comment:
Agree. I was too quick to fix the issue and did not look in the revised code.
![alt text][logo]
[logo]: https://pbs.twimg.com/profile_images/1526312969/unhappy.jpg "Gurzeh on Twitter"